astropy/uncertainty/tests/test_distribution.py

```diff
@@ -317,9 +317,9 @@ def test_reprs():
     ],
 )
 def test_wrong_kw_fails(func, kws):
+    kw_temp = kws.copy()
+    kw_temp["n_sample"] = 100  # note the missing "s"
```

Comment:
This could be written as
```suggestion
    kw_temp = kws | {"n_sample": 100}  # note the missing "s"
```

Reply:
Sounds good, will do it this way